tests/plugins/auto-closer/index.js

```diff
@@ -0,0 +1,199 @@
+
+describe("auto-closer", () => {
+    let bot;
+
+    beforeEach(async() => {
+        githubNock = nock("https://api.github.com");
+        bot = new probot.Application({
+            id: "test",
+            cert: "test",
+            cache: {
+                wrap: () => Promise.resolve({ data: { token: "test" } })
+            },
+            app: () => "test"
+        });
+
+        const { paginate } = await bot.auth();
+
+        bot.auth = () => Object.assign(new GitHubApi(), { paginate });
+
+        nock.disableNetConnect();
+
+        githubNock
+            .get("/app/installations")
+            .query(true)
+            .reply(200, [{
+                id: 1,
+                account: {
+                    login: "test"
+                }
+            }]);
+
+        githubNock
+            .get("/installation/repositories")
+            .query(true)
+            .reply(200, {
+                total_count: 1,
+                repositories: [
+                    {
+                        owner: {
+                            login: "test"
+                        },
+                        name: "repo-test"
+                    }
+                ]
+            });
+
+    });
+
+    afterEach(() => {
+        nock.cleanAll();
+    });
+
+    it("performs a search, and closes all returned issues", async() => {
+        const labelSearch = githubNock
+            .get("/repos/test/repo-test/labels")
+            .reply(200, [
+                {
+                    name: "auto closed"
+                },
+                {
+                    name: "accepted"
+                }
+            ]);
+
+        const acceptedIssueSearch = githubNock
+            .get("/search/issues")
+            .query(value => value.q.includes(" label:accepted"))
+            .reply(200, {
+                total_count: 2,
+                incomplete_results: false,
+                items: [
+                    { number: 1 },
+                    { number: 2 }
+                ]
+            }, {
+            });
+
+        const unacceptedIssueSearch = nock("https://api.github.com")
+            .get("/search/issues")
+            .query(value => value.q.includes(" -label:accepted"))
+            .reply(200, {
+                total_count: 2,
+                incomplete_results: false,
+                items: [
+                    { number: 3 },
+                    { number: 4 }
+                ]
+            }, {
+            });
+
+        // check that labels are updated
+        const issueNumbers = [1, 2, 3, 4];
+        const labelRequests = issueNumbers.map(issueNum => checkNockLabelRequest(issueNum));
+        const closedRequests = issueNumbers.map(issueNum => checkNockClosedRequest(issueNum));
+        const commentRequests = issueNumbers.map(issueNum => checkNockCommentRequest(issueNum));
+
+        autoCloser(bot);
+        await new Promise(resolve => setTimeout(resolve, 500));
```

Comment:
The unit test style needs to change. Please refer to my PR for scheduler unit test.